smriprep/cli/run.py

```diff
@@ -516,9 +527,11 @@ def build_workflow(opts, retval):
             "logging": {"log_directory": str(log_dir), "log_to_file": True},
             "execution": {
                 "crashdump_dir": str(log_dir),
-                "crashfile_format": "txt",
+                "crashfile_format": "pklz",
```

Comment:
what is the reason for this?

Reply:
Likewise, I need the debug and don't want to add flags/config values. We can revert before a release. Can maybe make a list of pre-merge todos?

Reply:
Sounds good